index.js

```diff
@@ -8,23 +8,24 @@ const core = require('@actions/core')
 
 module.exports = app => {
   app.on('push', async context => {
-    const config = await getConfig({
-      app,
-      context,
-      getConfig: require('probot-config')
-    })
+    let config
+    try {
+      config = await getConfig({
+        app,
+        context,
+        getConfig: require('probot-config')
+      })
+    } catch (error) {
+      log({ app, context, error, message: 'Invalid config file' })
+      return
+    }

```

Comment:
Would it be an idea to put the logging inside `getConfig` so that we can get rid of the `try/catch` here? That would make code a little more readable :)

Reply:
How would you exit early?

Reply:
You're right, this seems fine actually. Try/catch is just really noisy, not much we can do about that :)

Reply:
Oops you had already changed it. This is good!